.github/workflows/scripts/chart-integration/chart-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CHART_NAME=$1
+ROLLOUT_TIMEOUT_SECONDS=600s
+KUBECTL_TIMEOUT_SECONDS=60s
+RELEASE_NAME=${CHART_NAME}$(date +%Y%m%d%H%M%S)
+NAMESPACE=${CHART_NAME}-$(date +%Y%m%d%H%M%S)
+
+# step 1 Run init
+if [ -f ../../.github/workflows/scripts/chart-integration/${CHART_NAME}-init.sh ]; then
+    echo "Testing chart $CHART_NAME, init environment..."
+    ../../.github/workflows/scripts/chart-integration/${CHART_NAME}-init.sh
+    echo "Testing chart $CHART_NAME, init environment done!"
+fi
+
+# step 2 Helm install
+echo "Testing chart $CHART_NAME, installing chart with helm..."
+if helm install --create-namespace --namespace $NAMESPACE $RELEASE_NAME .; then
```

Comment:
you may use --wait and --timeout to wait for all resources is in ready status.

>  --wait                                       if set, will wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment, StatefulSet, or ReplicaSet are in a ready state before marking the release as successful. It will wait for as long as --timeout

Reply:
Thank you for comment. --wait is not a valid option. I use --timeout to set time limitation.

Reply:
Done